File: python/POO/lista1Ex5/classes/Pessoa.py
class Pessoa:
    def __init__(self):
        self.nome = ""
        self.cpf = 0
        self.idade = 0
        self.endereco = ""

    def __init__(self, nome="", cpf=0, idade=0, endereco=""):
        self.nome = nome
        self.cpf = cpf
        self.idade = idade
        self.endereco = endereco

    def __str__(self):
        return "Nome: {}\nCPF: {}\nIdade: {}\nEndereco: {}".format(self.nome, self.cpf, self.idade, self.endereco)

File: Lista2POO_GuilhermeHoerningReinert/lista2/src/ex1/Pessoa.java
package ex1;

public class Pessoa {
    private String nome;
    private int idade;
    private long cpf;
    private String cidade;

    public Pessoa() {
        this.nome = "";
        this.idade = 0;
        this.cpf = 0;
        this.cidade = "";
    }

    public Pessoa(String nome, int idade, long cpf, String cidade) {
        this.nome = nome;
        this.idade = idade;
        this.cpf = cpf;
        this.cidade = cidade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public int getIdade() {
        return idade;
    }

    public void setIdade(int idade) {
        this.idade = idade;
    }

    public long getCpf() {
        return cpf;
    }

    public void setCpf(long cpf) {
        this.cpf = cpf;
    }

    public String getCidade() {
        return cidade;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    @Override
    public String toString() {
        return nome + ", " + idade + ", " + cpf + ", " + cidade + '\n';
    }
}
